RUFAS/biophysical/animal/digestive_system/enteric_methane_calculator.py

```diff
@@ -0,0 +1,337 @@
+            ash_concentration = nutrient_concentrations["ash"]
+            soluble_residue = (
+                (100 - ash_concentration)
+                - neutral_detergent_fiber_concentration
+                - crude_protein_concentration
+                - ethyl_ester_concentration
+            )
+            gross_energy_concentration = (
+                0.263 * crude_protein_concentration
+                + 0.522 * ethyl_ester_concentration
+                + 0.198 * neutral_detergent_fiber_concentration
+                + 0.160 * soluble_residue
+            )  # [A.3B.C.2]
+            methane_emission = (0.065 * gross_energy_concentration * dry_matter_intake) / 0.05565  # [A.3B.C.3]
+
+        return methane_emission
+
+    @staticmethod
+    def calculate_cow_methane(
+        is_lactating: bool,
+        body_weight: float,
+        milk_fat: float,
+        metabolizable_energy_intake: float,
+        nutrient_amounts: dict[str, float],
+        nutrient_concentrations: dict[str, float],
+        methane_mitigation_method: str,
+        methane_mitigation_additive_amount: float,
+        methane_model: str,
+    ) -> float:
+        """
+        Calculates the daily enteric emissions for cows.
+
+        Parameters
+        ----------
+        body_weight: float
+            Body weight of the current cow (kg).
+        methane_model: str
+            Methane model used for methane emission calculations, including "Mutian", "Mills", "IPCC".
+        is_lactating: bool
+            Indicator of cow's lactating status.
+        milk_fat: float
+            Milk fat (from animal input), % of milk.
+        metabolizable_energy_intake: float
+            Metabolizable energy intake, Mcal/kg dry matter.
+        nutrient_amounts: Dict[str, float]
+            Amounts of nutrients in pen ration, calculated per animal, see Notes section for units.
+        nutrient_concentrations: Dict[str, float]
+             Concentrations of nutrients in pen ration, calculated per animal, percentages.
+        methane_mitigation_method: str
+            The name of the methane mitigation feed additives. The accepted names are
+                '3-NOP', 'Monensin', 'Essential Oils', and 'Seaweed'.
+        methane_mitigation_additive_amount: float
+            The dosage of the feed additive, mg/kg DMI.
+
+        Returns
+        -------
+        float
+            The daily enteric emissions for cows (g/day).
+
+            Notes
+            -----
+            nutrient_amount_units = {
+                "dm": "kg/animal",
+                "CP": "percent of DM",
+                "ADF": "percent of DM",
+                "NDF": "percent of DM",
+                "lignin": "percent of DM",
+                "ash": "percent of DM",
+                "phosphorus": "percent of DM",
+                "potassium": "percent of DM",
+                "N": "percent of DM",
+            }
```

Comment:
```suggestion
        Notes
        -----
        The dry matter ("dm") unit is kg per animal. Crude protein ("CP"), ADF, NDF, lignin, ash, phosphorus, potassium, and nitrogen ("N") are all percentages of dry matter.

```
I think it is not a good practice to put python in the docstring, unless it is in the Examples section